rpm_spec/manageiq.spec.in

```diff
@@ -69,11 +69,9 @@ cd %{_builddir}
 %{__mkdir} -p %{buildroot}/etc/httpd/conf.d
 %{__mkdir} -p %{buildroot}%{app_root}/log/apache
 %{__mkdir} -p %{buildroot}%{app_root}/tmp/{,sockets,pids}
-%{__mkdir} -p %{buildroot}%{app_root}/{certs,config}
-%{__mkdir} -p %{buildroot}%{app_root}/public/pictures
-%{__chmod} 4750 %{buildroot}%{app_root}/{log,config,certs}
+%{__chmod} 6750 %{buildroot}%{app_root}/{log,config,certs}
+%{__chmod} 6750 %{buildroot}%{app_root}/data/git_repos
```

Comment:
Would it be simpler to just 6750 the entire data dir into the previous line?
```suggestion
%{__chmod} 6750 %{buildroot}%{app_root}/{log,config,certs,data}
```

Reply:
I'm not sure what else is in there, but I assume anything else needs writability as well

Reply:
by default, the directories are `755`. So all the lines above changing to `manageiq.manageiq` makes them writeable and readable.

This change says that `root` (basically just `evmserverd` and `appliance_console`) will write to these directories, and we want `manageiq` to be able to read and write to those files. (the `umask` for root is 755 and makes our processes blow up.)

There area also some ui directories that need access. I have moved those over to the ui spec per previous request in this pr

Reply:
^ ignore.

So we can `chmod -r data` which will change anything that we add at a later time.
4

Reply:
I don't think we should `chmod -r`, because the content inside the git repos directory are external git repos.  Not sure what side effects that might cause.  Maybe @NickLaMuro has some thoughts here.

Reply:
So, seems that `git` will only track the executable permissions, not the full set of `nix` permission levels:

https://stackoverflow.com/questions/10516201/updating-and-committing-only-a-files-permissions-using-git-version-control

So, with that said, I assume what ever we do here, it needs to be able to allow anyone that executes playbooks that might come with a role (and I assume that could include a shell/python executable) to be able to run those scripts.  But that might be less of an issue since the repos here are just `bare` repos, and so the actually checkout happens into `/tmp`.

So we might be okay with whatever is done here as `/tmp` is where any actually interaction would come from from a potentially different and less privileged user.

Reply:
I chmod the data dir and the git_repo

let me know if this is not what you were asking for